apis/tempo/v1alpha1/tempomonolithic_defaults.go

```diff
@@ -13,9 +13,10 @@ import (
 )
 
 var (
-	twoGBQuantity           = resource.MustParse("2Gi")
-	tenGBQuantity           = resource.MustParse("10Gi")
-	defaultServicesDuration = metav1.Duration{Duration: time.Hour * 24 * 3}
+	twoGBQuantity                       = resource.MustParse("2Gi")
+	tenGBQuantity                       = resource.MustParse("10Gi")
+	defaultServicesDuration             = metav1.Duration{Duration: time.Hour * 24 * 3}
+	defaultFindTracesConcurrentRequests = 1
```

Comment:
I will keep the default to 1, here is some rationale: 

Customers complain when querying large number of traces e.g. 500-1500. A single trace get from S3 can take from 2s to 8s. for 500 traces it is 500*4=2000s/60=33mins. Cutting the time in half with 2 concurrent requests does not bring any value, however 30 concurrent requests significantly improve the situation but for that customers need to scale up queriers.

Reply:
I think we should use a default value which improves the current situation instead of leaving it as-is.

I like your suggestion of making the default value based on the number of queriers, maybe we can leave it `0` in the webhook and compute the default (`5 * .spec.template.querier.replicas` if set to `0`) in the manifest generation? I'm not sure what the appropriate amount of parallel requests per querier is, but imho it should handle more than 2 parallel requests per user/search request.

Reply:
My above comment is for TempoStack, for TempoMonolithic we can't scale up queriers, so maybe let's go with 2 if the default max concurent requests per querier is 20?

Reply:
I have changed the default for monolithic to 2 and tempostack to queriers*2.